modules/show_map.R

```diff
@@ -65,17 +66,19 @@ show_map_server <- function(id, com_module) {
     
     # Generate base map ----
     output$map <- renderLeaflet({
-      
-      data_snsrs <- try(get_locations()[[1]], silent = T)
-      shiny::validate(
 
-        need(class(data_snsrs) != "try-error", "Not yet selected any data.")
-      )
+      # data_snsrs <- try(get_locations()[[1]], silent = T) 
+      # print(data_snsrs)
+      # shiny::validate(
+      # 
+      #   need(class(data_snsrs) != "try-error", "Not yet selected any data.")
+      # )
+      #
+      #data_snsrs <- try(get_locations()[[1]], silent = T) %>% filter(., !grepl("KNMI|NL",station))
       
       ns("map")
       leaflet() %>%
-        #setView(5.384214, 52.153708 , zoom = 7) %>%
-        fitBounds(min(isolate(get_locations())[[2]]$lon), min(isolate(get_locations())[[2]]$lat), max(isolate(get_locations())[[2]]$lon), max(isolate(get_locations())[[2]]$lat)) %>% 
+        setView(5.384214, 52.153708 , zoom = 7) %>%
```

Comment:
I wouldn't hard-coded those values here, but in the global. So it is easy to change for others.